frontend/src/components/Arena/Arena.jsx

```diff
@@ -42,6 +55,14 @@ export default function Arena() {
       });
   };
 
+  const minute = () => {
+    return Math.floor(time / 60);
+  };
+
+  const second = () => {
+    return time % 60 < 10 ? `0${time % 60}` : time % 60;
```

Comment:
may be :

    const second = (time) => (time%60).toString().padStart(2, '0');

(and outside the component ;) )

https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/padStart